balrogscript/balrogscript/script.py

```diff
@@ -187,6 +188,26 @@ def submit_toplevel(task, config, auth0_secrets):
     ))
 
 
+# set_readonly {{{1
+def create_state_updater(**kwargs):
+    return ReleaseStateUpdater(**kwargs)
+
+
+def set_readonly(task, config, auth0_secrets):
+    state_updater = create_state_updater(
+        api_root=config['api_root'],
+        auth0_secrets=auth0_secrets,
+        dummy=config['dummy'],
+        suffix=task['payload'].get('blob_suffix', ''),
```

Comment:
Is this actually a feature we use, @MihaiTabara? If we don't, there's probably no reason to cargo cult it to this new run mode.

Reply:
We don't use this anymore AFAIK, nor I see any occurrence of it in-tree in the taskgraph.

```
mtabara@mozspace:[]~/work/mozilla/clones/hg/fxtree/taskcluster$ grep -R 'blob_suffix' * 
mtabara@mozspace:[]~/work/mozilla/clones/hg/fxtree/taskcluster$ 
```

Let's remove it indeed.